### PR TITLE
Navigation menu #3

### PR DESCRIPTION
**Pull request by Karl-J-Bruder**
New PR with fixes based on comments from previous PR.

### CODE REVIEW

File: src/components/atoms/MenuLink/index.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
 import { Link } from "gatsby"
```
Comment (Alfrex92) [inline src/components/atoms/MenuLink/index.tsx]: I think that you don't need to import any more `import { Link } from "gatsby"`

File: src/components/atoms/MenuLink/styles.tsx [hunk 2]
```diff
@@ -15,10 +15,13 @@ export const StyleNavMenuLink = styled.div`
...
 `;
 
+export const StyleLink = styled(Link)`
+    text-decoration: none;  
+`;
+
 export const StyleLinkName = styled.span`
     color: ${colors.gray.darker};
     font-size: ${fsize.bigger};
     margin: auto;
```
Comment (Alfrex92) [inline src/components/atoms/MenuLink/styles.tsx]: Are you using margin:auto to center something?

File: src/components/organisms/MobileNav/index.tsx [hunk 3]
```diff
@@ -1,28 +1,32 @@
 import React, { useState } from 'react'
+import data from "../../../../config/translations/en.json"
```
Comment (Alfrex92) [inline src/components/organisms/MobileNav/index.tsx]: `data` should come from props. What if the language is Japanese?

**Review by Alfrex92 — COMMENTED** [review]
Added some comments

File: src/components/atoms/Burger/index.tsx [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
 
-
-const BurgerIcon: React.FC<{ open: boolean, setOpen: Function }> = ({ open, setOpen }) => {
+const BurgerIcon: React.FC<{ open: boolean, setOpen: Function }> = (props) => {
     return (
-        <StyleBurger open={open} onClick={() => setOpen(!open)}>
-            <StyleDiv open={open} />
-            <StyleDiv open={open} />
-            <StyleDiv open={open} />
+        <StyleBurger open={props.open} onClick={() => props.setOpen(!props.open)}>
```
Comment (Alfrex92) [inline src/components/atoms/Burger/index.tsx]: I don't how to explain this but is safer to use 
`onClick={() => setOpen(oldProps => !oldProps)}`

Might be a good idea if you ask on stackoverflow or reactiflux the difference between:
`onClick={() => setOpen(oldProps => !oldProps)}`
and
`onClick={() => setOpen(!open)}`

File: src/components/atoms/MenuLink/styles.tsx (hunk 2 above)
Comment (Karl-J-Bruder) [reply src/components/atoms/MenuLink/styles.tsx]: I was using it to make sure that the link name would be centered within the StyleNavMenuLink div as well. If it's redundant, I can delete it.

File: src/components/organisms/MobileNav/index.tsx (hunk 3 above)
Comment (Karl-J-Bruder) [reply src/components/organisms/MobileNav/index.tsx]: I don't know. I can't figure out how to do that with imports.

File: src/components/atoms/Burger/index.tsx (hunk 4 above)
Comment (Karl-J-Bruder) [reply src/components/atoms/Burger/index.tsx]: Like this?

        <StyleBurger open={props.open} onClick={() => props.setOpen(!props.open)}>

File: src/components/atoms/MenuLink/styles.tsx (hunk 2 above)
Comment (Alfrex92) [reply src/components/atoms/MenuLink/styles.tsx]: In that case, I think that a safer approach will be `margin: 0 auto;` or `text-align:center`

File: src/components/atoms/Burger/index.tsx (hunk 4 above)
Comment (Alfrex92) [reply src/components/atoms/Burger/index.tsx]: Like this

`    <StyleBurger open={props.open} onClick={() => setOpen(oldProps => !oldProps)}`

`setOpen` is receiving the previous/old props as a value.

File: src/components/atoms/GalleryImage/GalleryImage.story.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from "react"
```
Comment (Alfrex92) [inline src/components/atoms/GalleryImage/GalleryImage.story.tsx:1]: You're committing these changes (Gallery) to this PR. Gallery and Navigation Menu are different components and each one needs to have their own PR.  I can approve it, but at work, this PR will be rejected.

**Review by Alfrex92 — COMMENTED** [review]
I left some comments.

Before you make the changes don't forget to do  ` git pull origin develop` since I just merged `MemberInfo` to develop branch.

File: src/components/organisms/MobileNav/index.tsx (hunk 3 above)
Comment (Alfrex92) [reply src/components/organisms/MobileNav/index.tsx]: Check my previous PR 😄 
https://github.com/Alfrex92/animers/pull/4

File: src/components/atoms/MenuLink/styles.tsx (hunk 2 above)
Comment (Karl-J-Bruder) [reply src/components/atoms/MenuLink/styles.tsx]: Okay, I went with text-align:center. Look fine now.

File: src/components/organisms/MobileNav/index.tsx (hunk 3 above)
Comment (Karl-J-Bruder) [reply src/components/organisms/MobileNav/index.tsx]: Okay, I think I got it this time lol

**Comment by Alfrex92** [comment]
Ok you can merge now this PR. Just don't forget to do 
` git pull origin develop` and then push your changes.

**Pull request merged**